### PR TITLE
[BN] updated document for version 1.3.0

### PR DESCRIPTION
**Pull request by m4hi2**
- removed white space from the end of line
- fixed missing reference links
- fixed spelling misktakes
- adopted some parts to match the latest EN document

### CODE REVIEW

File: lessons/bn/basics/basics.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 ---
 
-## ইন্সটলেশন ও ইন্টারএ্যাক্টিভ মোড
+## শুরুর কথা
+## এলিক্সির ইন্সটলেশন
 
-### ইন্সটলেশন
 
-শুরুতেই আমাদের এলিক্সির ইন্সটল করতে হবে । আলাদা আলাদা অপারেটিং সিস্টেমের জন্য ইন্সটলেশনের বিস্তারিত নির্দেশনা পাওয়া যাবে এলিক্সির এর অফিশিয়াল ওয়েব সাইটের এই সেকশনে - [Installing Elixir](http://elixir-lang.org/install.html) । এই গাইডলাইন অনুসরন করে প্রথমে এলিক্সির ইন্সটল করে নিন । 
+শুরুতেই আমাদের এলিক্সির ইন্সটল করতে হবে । আলাদা আলাদা অপারেটিং সিস্টেমের জন্য ইন্সটলেশনের বিস্তারিত নির্দেশনা পাওয়া যাবে এলিক্সির এর অফিশিয়াল ওয়েব সাইটের এই সেকশনে - [Installing Elixir](http://elixir-lang.org/install.html) । এই গাইডলাইন অনুসরন করে প্রথমে এলিক্সির ইন্সটল করে নিন ।
```
Comment (wasi0013) [inline lessons/bn/basics/basics.md:14]: Welcome to elixirschool, @m4hi2 bhai! Just nitpicking can you also remove the extra space in between the word and the Bangla full stop symbol? 
For example: `হবে ।` should be `হবে।` 
There is also a small spelling mistake: `অনুসরন` should be `অনুসরণ` as far as I know. 
Thanks for your valuable contribution! 😄

File: lessons/bn/basics/basics.md [hunk 2]
```diff
@@ -22,15 +22,17 @@
...
+ইন্টারএ্যাক্টিভ শেল চালু করতে কমান্ডলাইনে টাইপ করুন - `iex`:
 
     Erlang/OTP {{ site.erlang.OTP }} [erts-{{ site.erlang.erts }}] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]
 
     Interactive Elixir ({{ site.elixir.version }}) - press Ctrl+C to exit (type h() ENTER for help)
     iex>
 
+বি:দ্র: উন্ডোজপাওয়ার শেলে আপনাকে `iex.bat` টাইপ করতে হবে।
+
 আসুন আমরা কিছু এক্সপ্রেশন টাইপ করি এবং সেগুলোর আউটপুট দেখে নেই :
```
Comment (wasi0013) [inline lessons/bn/basics/basics.md:36]: The symbol `:` can be replaced with the Bengali version `ঃ`
Example: `নেই :` should be `নেইঃ`

File: lessons/bn/basics/basics.md [hunk 3]
```diff
@@ -213,9 +213,11 @@ iex> not 42
 ** (ArgumentError) argument error
 ```
 
-### কম্প্যারিজন
+বি:দ্র: এলিক্সির এর `and` এবং `or` আসলে আর্ল্যাং এর `andalso` এবং `orelse` অপারেটরে ম্যাপ করা।
+
+### তুলনা
 
-এলিক্সির গতানুগতিক সব কম্প্যারিজন অপারেটর সাপোর্ট করে : `==`, `!=`, `===`, `!==`, `<=`, `>=`, `<` এবং `>`.
+এলিক্সির গতানুগতিক সব কম্প্যারিজন(তুলনামূলক) অপারেটর সাপোর্ট করে : `==`, `!=`, `===`, `!==`, `<=`, `>=`, `<` এবং `>`.
```
Comment (wasi0013) [inline lessons/bn/basics/basics.md:220]: We can replace the `.` with Bengali full stop `।`

**Review by wasi0013 — CHANGES_REQUESTED** [review]
Excellent work!

**Comment by doomspork** [comment]
Thank you and welcome @m4hi2! Thank you for reviewing @wasi0013 ❤️ If it would be easier to review with a proper team I'm happy to set that up for you 🚀

File: lessons/bn/basics/basics.md (hunk 1 above)
Comment (m4hi2) [reply lessons/bn/basics/basics.md:14]: @wasi0013 bhai, thanks for the suggestions. these are dinosaurs from past localization efforts. I'll fix these and be mindful of issues like this in the future. Thanks again.

**Comment by wasi0013** [comment]
> Thank you and welcome @m4hi2! Thank you for reviewing @wasi0013 :heart: If it would be easier to review with a proper team I'm happy to set that up for you :rocket:

Thanks, @doomspork! That would be great! If it's possible you can also add @m4hi2 & @code-shoily to the team. They are my friends and mentor :)

File: lessons/bn/basics/basics.md (hunk 3 above)
Comment (m4hi2) [reply lessons/bn/basics/basics.md]: fixed.

File: lessons/bn/basics/basics.md (hunk 2 above)
Comment (m4hi2) [reply lessons/bn/basics/basics.md]: @wasi0013 bhai yes, traditionally বিঃদ্রঃ is written, so I think we should go that route. waiting for your confirmation.

**Review by wasi0013 — APPROVED** [review]
Thanks, @m4hi2 bhai.

**Comment by doomspork** [comment]
Thank you @m4hi2 and @wasi0013! I created the @elixirschool/bengali team and invited you three 🚀

**Pull request merged**